src/api/sendRequest.js

```diff
@@ -95,10 +92,7 @@ const sendRequest = ({
     }
     return res.json().then((json) => {
       if (json.code === 403) {
```

Comment:
In case of a 403, can the body of response be displayed as a snackbar?

Reply:
Yes, we could do that but then we would not be able to log out the user on error 403.
As logout will redirect to the home page before the message is displayed.

Reply:
Is it possible to store the error code as a state somewhere and pass it to home?
And then home reads it and shows a snackbar? If the state does not exist, then show nothing...

I am not sure how this can be done in react. May be pass it as a GET value during redirect?

Reply:
@GMishx I was trying to find the best way to do that but currently, it seems that then we have to move the check for 403 error code from `sendRequest.js` file to all the places where API call actually happens. As the error information can only be passed from react jsx file and it's not possible to provide that info through simple javascript code.

Should I update the code to check for the 403 error code while making the request?

Reply:
As far as I understood, every call is routed by `sendRequest.js` which checks for 403 and calls `logout()`. Now logout redirect user to home page after deleting the cookie.
How about adding a `?error="<response body>"` to this redirect and reading this in the home page. If `error` exists in the `GET` request, read it, show a snackbar and remove it from the address bar.

Does this sounds good?

Reply:
Yeah sure, if we have no issue in passing error data through URL (as it would be visible to the user) then we could do that.

Reply:
Yeah it is fine. I have seen many sites doing that.
Otherwise, there can also be a local store variable doing that if you are not comfortable sharing it over URL.

And would you like to implement it here or in a separate PR?

Reply:
> And would you like to implement it here or in a separate PR?

Anything works for me. Though a separate one would be better for tracking as it as the new feature does not goes well with the PR title. What would you suggest?